composer.json

```diff
@@ -23,26 +23,26 @@
   ],
   "require": {
     "php": "^7.2",
-    "ext-imagick" : "*",
-    "illuminate/bus": "~5.5.0|~5.6.0|~5.7.0|~5.8.0",
-    "illuminate/console": "~5.5.0|~5.6.0|~5.7.0|~5.8.0",
-    "illuminate/database": "~5.5.0|~5.6.0|~5.7.0|~5.8.0",
-    "illuminate/pipeline": "~5.5.0|~5.6.0|~5.7.0|~5.8.0",
-    "illuminate/support": "~5.5.0|~5.6.0|~5.7.0|~5.8.0",
+    "ext-fileinfo": "*",
+    "ext-imagick": "*",
+    "illuminate/bus": "^6.0",
```

Comment:
Can't we allow L5.5+ here?

Reply:
We can, I've done it like in most others packages were only the latest release is supported. And because of the different versioning system I beliefe that this would be a good move.
Primary because L5.5 looses feature support in 3 days.